packages/backend/__mocks__/@podman-desktop/api.js

```diff
@@ -0,0 +1,18 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+export {};
```

Comment:
FYI @deboer-tim reason for this is because internally macadam.js also calls @podman-desktop/api.js and it was recommended to add a mocks functionality here to prevent conflicts.

Without it, it'll try and import @podman-desktop when referencing @crc-org/macadam.js within the tests.